cfme/infrastructure/virtual_machines.py

```diff
@@ -353,40 +353,58 @@ def _nav_to_snapshot_mgmt(self):
         def does_snapshot_exist(self):
             self._nav_to_snapshot_mgmt()
             try:
-                if self.name is not None:
-                    self.snapshot_tree.find_path_to(re.compile(self.name + r".*?"))
-                else:
-                    self.snapshot_tree.find_path_to(re.compile(self.description + r".*?"))
+                self.snapshot_tree.find_path_to(
+                    re.compile(r"{}.*?".format(self.name or self.description)))
                 return True
             except CandidateNotFound:
                 return False
             except NoSuchElementException:
                 return False
 
-        def wait_for_snapshot_active(self):
-            self._nav_to_snapshot_mgmt()
+        def _snapshot_click_helper(self, prop):
+            """Helper method to reduce code duplication.
```

Comment:
This description is too generic (most functions/methods are written to reduce code duplication). We also don't use `_helper` postfixes and `snapshot` is also not necessary in the method's name, since it's a method of the `Snapshot` class. `_click_tree_path` or something similar is more than enough.